docker-compose.yml

```diff
@@ -14,9 +29,11 @@ services:
       - ./migrations/dlu:/docker-entrypoint-initdb.d
```

Comment:
Hm, maybe we now should also make a Dockerfile for the container which includes the migrations. Then we are able to distribute just the docker-compose.yml later down the road. But you don't need to change this right now. This is just a thought.

Reply:
Actually there's no need for that since this just uses the stock MariaDB image. The build/deploy time is only a couple of seconds.